apisix/plugins/gzip.lua

```diff
@@ -21,17 +21,26 @@ local req_http_version = ngx.req.http_version
 local str_sub = string.sub
 local ipairs = ipairs
 local tonumber = tonumber
+local type = type
 
 
 local schema = {
     type = "object",
     properties = {
         types = {
-            type = "array",
-            minItems = 1,
-            items = {
-                type = "string",
-                minLength = 1,
+            anyOf = {
+                {
+                    type = "array",
+                    minItem = 1,
+                    items = {
+                        type = "string",
+                        minLength = 1,
+                    },
+                },
+                {
+                    type = "string",
```

Comment:
I think `enum = {"*"}` is enough?

Reply:
@spacewander Yes, I think it's simpler to use {"*"}, this pr only try to follow nginx's config strictly.
If that not import, I will change it to {"*"}

Reply:
misunderstood you firstly, now i got it.